src/index.html

```diff
@@ -9,6 +9,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="logo-img"
+          src="./images/logo.png"
+          alt="logo-icon"
+        >
+      </a>
+      <nav class="nav-list">
+        <ul class="list-header-menu ">
+          <li class="element-header-menu">
+            <a href="apple" class="is-active link">Apple</a>
+          </li>
+
+          <li class="element-header-menu">
+            <a href="samsung" class="link">Samsung</a>
+          </li>
+
+          <li class="element-header-menu">
+            <a href="smartphones" class="link">Smartphones</a>
+          </li>
+
+          <li class="element-header-menu">
+            <a
+              data-qa="hover"
+              href="#"
+              class="link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="element-header-menu">
+            <a href="#" class="link">Gadgets</a>
```

Comment:
Let's make all links more informative
```suggestion
            <a href="#gadgets" class="link">Gadgets</a>
```